### ISSUE TITLE
`print_model_weights` fails for multi-class logistic regression models

### ISSUE THREAD

**Issue by desilinguist** [EducationalTestingService/skll#274]
```
Traceback (most recent call last):
  File "/Users/nmadnani/anaconda/envs/rsmtool/bin/print_model_weights", line 6, in <module>
    sys.exit(main())
  File "/Users/nmadnani/anaconda/envs/rsmtool/lib/python3.4/site-packages/skll/utilities/print_model_weights.py", line 56, in main
    (weights, intercept) = learner.model_params
  File "/Users/nmadnani/anaconda/envs/rsmtool/lib/python3.4/site-packages/skll/learner.py", line 712, in model_params
    if self.model.intercept_:
ValueError: The truth value of an array with more than one element is ambiguous. Use a.any() or a.all()
```

It fails because `self.model.intercept_` is an array and, as the error says, you can't treat an array as a `bool`. We don't have a test for this case - we [check](https://github.com/EducationalTestingService/skll/blob/master/tests/test_utilities.py#L368) binary `LogisticRegression`, 2-class `LinearRegression`, and multi-class `LinearSVR` but not multi-class `LogisticRegression`. It's still surprising that no one else has run into this error though.

**Comment by desilinguist** [comment]
this will be addressed by #267

**Comment by desilinguist** [comment]
Addressed by #267.
